pyfar/dsp/dsp.py

```diff
@@ -811,6 +813,338 @@ def regularized_spectrum_inversion(
+        ``'magnitude_linear'``
+            Interpolation of the magnitude values and generation of a linear
+            phase response with a specified group delay. (See parameters
+            `group_delay` and `unit` below and
+            :py:func:`~pyfar.dsp.linear_phase`).
+        ``'magnitude_minimum'``
+            Interpolation of the magnitude values and generation of a minimum
+            phase response
+        ``'magnitude'``
+            Interpolate the magnitude values only. Results in a zero phase
+            signal, which is symmetric around the first sample
+
+    kind : tuple
+        Three element tuple ``('first', 'second', 'third')`` that specifies the
+        kind of inter/extrapolation below the lowest frequency (first), between
+        the lowest and highest frequency (second), and above the highest
+        frequency (third).
+
+        The string has to be ``'linear'``, ``'nearest'``, ``'nearest-up'``,
+        ``'zero'``, ``'slinear'``, ``'quadratic'``, ``'cubic'``,
+        ``'previous'``, or ``'next'``.  ``'zero'``, ``slinear``,
+        ``'quadratic'``, and ``'cubic'`` refer to a spline interpolation of
+        zeroth, first, second or third order; ``'previous'`` and ``'next'``
+        simply return the previous or next value of the point; ``'nearest-up'``
+        and ``'nearest'`` differ when interpolating half-integers
+        (e.g. 0.5, 1.5) in that ``'nearest-up'`` rounds up and ``'nearest'``
+        rounds down. The interpolation is done using
+        ``scipy.interpolate.interp1d``.
+    fscale : string, optional
+
+        ``'linear'``
+            Interpolate on a linear frequency axis.
+        ``'log'``
+            Interpolate on a logarithmic frequency axis. Note that 0 Hz can
+            not be interpolated on a logarithmic scale because the logarithm
+            of 0 does not exist. Frequencies of 0 Hz are thus replaced by the
+            next highest frequency before interpolation.
+
+        The default is ``'linear'``.
+    clip : bool, tuple
+        The interpolated magnitude response is clipped to the range specified
+        by this two element tuple. E.g., ``clip=(0, 1)`` will assure that no
+        values smaller than 0 and larger than 1 occur in the interpolated
+        magnitude response. The clipping is applied after the interpolation
+        but before applying linear or minimum phase (in case `method` is
+        ``'magnitude_linear'`` or ``'magnitude_minimum'``. The default is
+        ``False`` which does not clip the
+        data.
+    group_delay : float, optional
+        The group delay of the linear phase response. Must be specified if
+        `method` is ``'magnitude_linear'``. The unit of the group delay is
+        given by `unit` (see below).
+    unit : string, optional
+        Unit of the group delay. Can be ``'s'`` for seconds, ``'ms'`` for
+        milliseconds, ``'mus'`` for microseconds, or ``'samples'``. The
+        default is ``'samples'``.
```

Comment:
These could be removed following my comment above

Reply:
yep :)